core/src/main/java/hudson/PluginManager.java

```diff
@@ -948,52 +948,50 @@ public void dynamicLoad(File arc, boolean removeExisting, @CheckForNull List<Plu
 
     @Restricted(NoExternalUse.class)
     public void start(List<PluginWrapper> plugins) throws Exception {
-      try (ACLContext context = ACL.as(ACL.SYSTEM)) {
```

Comment:
For other reviewers: **Hide whitespace changes**

Reply:
Yes, there were too many whitespaces inconsistencies in this method, I ended up reformatting it.

Reply:
https://github.com/jenkinsci/jenkins/pull/4393/files?w=1